ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -238,6 +275,7 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
             textBlock.FontSize = _fontSize ?? 15;
             textBlock.FontStyle = _fontStyle ?? new FontStyle();
             textBlock.FontWeight = _fontWeight ?? FontWeights.Normal;
+            textBlock.TextDecorations = new TextDecorationCollection(TextDecorationCollection);
 
```

Comment:
This would be supported on Android and iOS:
```
<Text>
   Here is some regular text <Text style={{textDecorationLine: 'underline'}}>and here is some underlined virtual text</Text>.
</Text>
```
However, this scenario would not supported by this PR.